docs/io/fits/index.rst

```diff
@@ -473,6 +473,42 @@ table's structure needs more information. First of all, tables can only be an
 extension HDU, not a primary. There are two kinds of FITS table extensions:
 ASCII and binary. We'll use binary table examples here.
 
+.. note::
+    
+    If you want to create a simple binary FITS table with no other HDUs,
+    you should use the Table_ interface in Astropy instead and then 
+    write to FITS.
+
+    >>> from astropy.table import Table
+    >>> t = Table([[1,2],[4,5],[7,8]], names=('a','b','c'))
+    >>> t.write('table.fits', format='fits')
+
+    Same code in PyFITS would look like this:
+
+    >>> import pyfits
+    >>> import numpy as np
+    >>> c1 = np.array([1,2])
+    >>> c2 = np.array([4,5])
+    >>> c3 = np.array([7,8])
+    >>> t = pyfits.BinTableHDU.from_columns([
+    ...     pyfits.Column(name='a', array=c1),
+    ...     pyfits.Column(name='b', array=c2),
+    ...     pyfits.Column(name='c', array=c3)])
```

Comment:
This doesn't work, a format is required, something like `fits.Column(name='a', array=[1,2], format='K')`.